runtime/plaid/src/executor/mod.rs

```diff
@@ -437,11 +451,29 @@ fn process_message_with_module(
                         format!("{}_computation_percentage_used", module.name),
                         computation_used as i64,
                     )?;
+
+                    // If benchmarking is enabled, log data to the benchmarking system
+                    if let Some(ref sender) = benchmark_mode {
+                        if let Err(e) = sender.blocking_send(ModulePerformanceMetadata {
```

Comment:
Why is this a blocking send when send everywhere else has been fine?

Reply:
The channel we use for the benchmarking system is the `tokio::sync::mpsc` one instead of `crossbeam_channel`. Can switch to crossbeam and use `recv_timeout` in the benchmarking loop instead